llvm/test/CodeGen/X86/code-model-elf.ll

```diff
@@ -385,8 +385,8 @@ define dso_local i32 @load_forced_small_data() #0 {
 ;
 ; MEDIUM-STATIC-LABEL: load_forced_small_data:
 ; MEDIUM-STATIC:       # %bb.0:
-; MEDIUM-STATIC-NEXT:    movl $forced_small_data, %eax
-; MEDIUM-STATIC-NEXT:    movl 8(%rax), %eax
+; MEDIUM-STATIC-NEXT:    movl $forced_small_data+8, %eax
```

Comment:
The TODO above still seems relevant, and I think to achieve that, we would need to do a RIP-relative access in the medium static model, which is out of scope for this change, that's in the code which controls WrapperRIP emission.